include/circt/Dialect/FIRRTL/Passes.td

```diff
@@ -117,7 +117,7 @@ def RemoveUnusedPorts : Pass<"firrtl-remove-unused-ports", "firrtl::CircuitOp">
   ];
 }
 
-def IMDeadCodeElim : Pass<"firrtl-imdeadcodeelim", "firrtl::CircuitOp"> {
```

Comment:
I think hierpath are now in the circuit namespace.

Reply:
I followed the implementation of InnerSymbolDCE that checks all uses of inner symbols not only within circuit op.

https://github.com/llvm/circt/blob/a6c8cb3b14ae1b3f31ce28262c2be54b6decac57/lib/Dialect/FIRRTL/Transforms/InnerSymbolDCE.cpp#L107-L109 

If it's assumed that there is no user of inner symbol outer circuit op, it makes sense to me to make these passes circuit op.